curriculum/test/utils/mongo-ids.js

```diff
@@ -1521,7 +1521,10 @@ const duplicatedProjectIds = [
   /*** Legacy Only ***/
 
   // Caesars Cipher
-  '56533eb9ac21ba0edf2244e2'
+  '56533eb9ac21ba0edf2244e2',
+
+  // Polygon area calculator
+  '5e444147903586ffb414c94f'
```

Comment:
Can we move this below the budget app?

Also, I know that it was not added in this PR, but why do we have the Caesar cipher here?

Reply:
This Caesar Cipher is part of the legacy [JS cert](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/javascript-algorithms-and-data-structures-projects/caesars-cipher).

I have changed the order for the polygon area calculator.

Reply:
> This Caesar Cipher is part of the legacy [JS cert](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/javascript-algorithms-and-data-structures-projects/caesars-cipher).

But I don't see it repeated anywhere. And [the one we have for Python](https://github.com/freeCodeCamp/freeCodeCamp/pull/60198) is a different implementation (step-based also). Am I missing something?